src/include/dhcp-server/widgets.rb

```diff
@@ -1054,15 +1009,19 @@ def InitWidgets
           #FIXME CWM should be able to handle virtual widgets
           "widget"        => :textentry
         },
-        "start"                => {
-          "widget" => :checkbox,
-          # check box
-          "label"  => _("&Start DHCP Server"),
-          "help"   => Ops.get(@HELPS, "start", ""),
-          "init"   => fun_ref(method(:startInit), "void (string)"),
-          "handle" => fun_ref(method(:startHandle), "symbol (string, map)"),
-          "store"  => fun_ref(method(:startStore), "void (string, map)"),
-          "opt"    => [:notify]
+        "service_status"         => {
+          "widget" => :custom,
+          "custom_widget" => @status_widget.widget,
+          "help"   => @status_widget.help,
+          "init"   => fun_ref(method(:init_service_status), "void (string)"),
+          "handle" => fun_ref(method(:handle_service_status), "symbol (string, map)"),
+          "store"  => fun_ref(method(:store_service_status), "void (string, map)")
+        },
+        "apply"           => {
+          "widget" => :custom,
+          "custom_widget" => PushButton(Id("apply"), _("Apply Changes")),
+          "handle" => fun_ref(method(:handle_apply), "symbol (string, map)"),
```

Comment:
well if you use handle for specific widget, then maybe it make sense to not check event_id at all, as it should be called only for this button